cors.go

```diff
@@ -22,6 +23,7 @@ const (
 
 type Config struct {
 	AllowedOrigins   []string
+	OriginRegexp     *regexp.Regexp
```

Comment:
If you can specify `origin` multiple times, you should probably be allowed to supply `origin_regex` multiple times. Can we make this a slice?